scripts/gen_profiles_layer.py

```diff
@@ -2943,7 +2943,7 @@ class VulkanProfilesLayerGenerator():
     additional_properties = ['VkPhysicalDeviceProperties', 'VkPhysicalDeviceLimits', 'VkPhysicalDeviceSparseProperties', 'VkPhysicalDeviceToolProperties', 'VkPhysicalDevicePortabilitySubsetPropertiesKHR']
     # VkPhysicalDeviceHostImageCopyFeaturesEXT is not ignored to allow the people using the MockICD to still have the feature enabled,
     # but use the properties in the MockICD until HostImageCopyPropertiesEXT is fixed.
-    ignored_structs = ['VkPhysicalDeviceHostImageCopyPropertiesEXT']
+    ignored_structs = ['VkPhysicalDeviceHostImageCopyPropertiesEXT', 'VkPhysicalDeviceLayeredApiPropertiesListKHR']
```

Comment:
@christophe-lunarg Another struct we are ignoring, because it has a pointer in it